RF24Network_config.h

```diff
@@ -55,7 +55,11 @@
      * the actual transmitted payload to 24 bytes (which is also the default behavior on ATTiny devices).
      */
     #ifndef MAX_PAYLOAD_SIZE
-        #define MAX_PAYLOAD_SIZE 144
+        #if defined linux || defined __linux
+            #define MAX_PAYLOAD_SIZE 1514
```

Comment:
1. How did you arrive at this number? With 144, I guessed that 6 consecutive transmissions was a safe default.
2. Does this result in a higher risk of collision?

Reply:
Shouldn't it be a multiple of 24 bytes (for optimal fragmentation)?

Reply:
The 1514 seems to be the right size to allow a standard MTU of 1500 bytes when using RF24Gateway/Ethernet with a TAP interface. The link layer header length is 14, so 1514 allows MTU of 1500. The 144 was due to being a multiple of 24.

I think your correct on the other points, we should leave it as-is for other platforms.

Reply:
Also, yes, a higher MAX_PAYLOAD_SIZE can result in a higher chance of collisions, due to the longer transmission times, but surprisingly it works OK due to the retries, timeout periods etc used, even with a large network.